docs/en/notes/faq.md

```diff
@@ -26,5 +26,86 @@ Notes:
 
```

Comment:
please add notes according to zh_cn faq
https://github.com/open-mmlab/mmsegmentation/blob/1896344e04ad3a919194b394d44da4e0b69d026b/docs/zh_cn/notes/faq.md?plain=1#L21-L25